src/supernova/mod.rs

```diff
@@ -446,12 +459,12 @@ where
         scalar_as_base::<G1>(pp.digest()),
         G1::Scalar::ZERO,
         z0_primary,
-        None,
-        None,
-        None,
-        None,
-        Some(G1::Scalar::from(circuit_index as u64)),
-        G1::Scalar::ZERO, // set augmented circuit index selector to 0 in base case
+        None,                                         // zi = None for basecase
```

Comment:
Weird spacing.

Reply:
Added an auxiliary variable to make `rustfmt` play nice.